PrestaShop/PrestaShop
# Country names from CLDR

## Description

While checking out CLDR I saw that it contains localized country names as well.
PrestaShop is missing a lot of translations so CLDR also seems to be useful here for country names.
I made a quick 'n dirty Node.js script to generate the translation files: https://gist.github.com/firstred/0b47dac668a520378c788859767a672d
## Steps to Test this Feature

Step 1. Go to Duolingo
Step 2. Complete every course
Step 3. With your newly gained language skills, check all the country names
OR
Step 1. Check out the gist used for the generation of the files: https://gist.github.com/firstred/0b47dac668a520378c788859767a672d
Step 2. Check if the language codes are mapped correctly
Step 3. Check if all files are correct

Whichever you prefer 😜
